moncli/types.py

```diff
@@ -125,6 +127,45 @@ class TextType(MondayType):
     null_value = ""
 
 
+class DateType(MondayType):
+
+    native_type = datetime
+    allow_casts = (str, int)
+    null_value = {}
+    has_time = False
+
+    def __init__(self, id: str = None, title: str = None, has_time: bool = False, *args, **kwargs):
+        self.id = id
+        self.title = title
+        self.has_time = has_time
+        super().__init__(self.id, self.title, *args, **kwargs)
+
+    def _cast(self, value):
+        if isinstance(value, int):
+            try:
+                return datetime.fromtimestamp(value)
+            except ValueError:
+                raise ConversionError('Invalid UNIX timestamp "{}".'.format(value))
+        if isinstance(value, str):
+            try:
+                date = datetime.strptime(value.split(' ', 1)[0], DATE_FORMAT)
+            except ValueError:
+                raise ConversionError('Cannot convert value "{}" into Date.'.format(value))
+            if self.has_time == True:
```

Comment:
This situation handles when we have a time.  However, when has_time is False, there is no return value (None) and it blows up _export.  Please return the _date_ if has_time is False.